nobl9/README.md

```diff
@@ -0,0 +1,59 @@
+# Nobl9
+
+## Overview
+Nobl9 is an SLO platform that provides real-time and historical SLO reports.
+Nobl9 integrates with Datadog to collect SLI metrics and measure them against
+SLO targets. Since Nobl9 calculates error budgets of acceptable thresholds, it
+can trigger workflows and alerts when the error burn rate is too high or has
+been exceeded.
```

Comment:
```suggestion
Nobl9 is a SLO platform that provides real-time, historical SLO reports. Nobl9 integrates with Datadog to collect SLI metrics and measure them against SLO targets. Since Nobl9 calculates error budgets of acceptable thresholds, it can trigger workflows and alerts when the error burn rate is too high or has been exceeded.
```